service-base/src/main/java/org/eclipse/hono/service/amqp/AmqpEndpoint.java

```diff
@@ -42,4 +42,11 @@
      * @param sourceAddress The (remote) source address from the client's AMQP <em>ATTACH</em> message.
      */
     void onLinkAttach(ProtonConnection connection, ProtonSender sender, ResourceIdentifier sourceAddress);
+
+    /**
+     * Handles a lost connection.
```

Comment:
IMHO it would be helpful if we could describe in more detail under which circumstances this method will be invoked.